modules/canbus/vehicle/lexus/protocol/accel_cmd_100.cc

```diff
@@ -123,7 +123,9 @@ Accelcmd100* Accelcmd100::set_accel_cmd(double accel_cmd) {
 // 'len': 16, 'is_signed_var': False, 'physical_range': '[0|1]', 'bit': 15,
 // 'type': 'double', 'order': 'motorola', 'physical_unit': ''}
 void Accelcmd100::set_p_accel_cmd(uint8_t* data, double accel_cmd) {
-  accel_cmd = std::max(0.0, (accel_cmd - 8.0) / 100);
+  const double scaling_bias = 7.853;   // estimated from the garage test data
+  const double scaling_slope = 1.176;  // estimated from the garage test data
```

Comment:
maybe use `scaling_gain` to avoid confusion?

Reply:
get it